src/coreclr/src/vm/customattribute.cpp

```diff
@@ -937,6 +936,7 @@ FCIMPL7(void, COMCustomAttribute::GetPropertyOrFieldData, ReflectModuleBaseObjec
                 break;
             case SERIALIZATION_TYPE_SZARRAY:
             {
+                *value = NULL;
```

Comment:
Nit: Should this be moved above the _switch_ statement to make this code more future-proof should new case statements be added in the future?

Reply:
Was considered, but there's the tiny cost of double assignment.